sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified_comments.json

```diff
@@ -2504,28 +2499,28 @@
           "type": "integer"
         },
         "deviceEtag": {
-          "description": "Device's ETag",
+          "description": "TODO - which operation does this affect?",
           "type": "string"
         },
         "status": {
-          "description": "Gets the corresponding Device's Status.",
+          "description": "Gets the device's status.",
           "enum": [
             "enabled",
             "disabled"
           ],
           "type": "string"
         },
         "statusReason": {
-          "description": "Reason, if any, for the corresponding Device to be in specified Status",
+          "description": "Reason, if any, for the device to be in specified status.",
```

Comment:
"Reason for current device status"?

Reply:
Rephrased to: `Reason, if any, for the device's current status.`